Sources/Subprocess/Platforms/Subprocess+Linux.swift

```diff
@@ -191,7 +191,7 @@ extension Configuration {
 public struct ProcessIdentifier: Sendable, Hashable {
     /// The platform specific process identifier value
     public let value: pid_t
-    internal let processDescriptor: PlatformFileDescriptor
+    public let processDescriptor: CInt
```

Comment:
nit: since we are using it else where might as well keep it here as well

Reply:
`PlatformFileDescriptor` is internal